src/Symfony/Component/Mailer/Transport/SendmailTransport.php

```diff
@@ -59,6 +60,8 @@ public function __construct(string $command = null, EventDispatcherInterface $di
             }
 
             $this->command = $command;
+        } else {
+            $this->command = \ini_get('sendmail_path') ?: self::SENDMAIL_FHS_PATH;
```

Comment:
This will change the default behavior: before: `-bs`, after `-t -i` on my Ubuntu.
We need another way to opt-in for reading from ini